### ISSUE TITLE
maybe a "Apple Maps" problem ?

### ISSUE THREAD

**Issue by bau720123** [dpa99c/phonegap-launch-navigator#26]
hi @dpa99c 
I am using cordova cli 5.1.1 + your latest version from npm +ipad mini 8.4.1(12H321)
well...this is my full code

```
function checkLaunchNavigator2(action, action2)
{
var options = { preferGoogleMaps: false, transportMode: "driving", enableDebug: false, disableAutoGeolocation: false }

launchnavigator.navigate(action, action2,
 function()
 {
 console.log('success');
 },
 function(error)
 {
 console.log('fail');
 },
 options
);
}

<a onClick="checkLaunchNavigator2('London, UK', 'Manchester, UK');">STEP1</a>
<a onClick="checkLaunchNavigator2([50.279306, -5.163158], [50.342847, -4.749904]);">STEP2</a>
```

first time，when you use "STEP1"，it will show the Apple Maps and have a title "Manchester to London" 

     don't close the Apple Maps and change back to your app...

second time，when you click "STEP2"，it will show the Apple Maps and have a title "Satrt to Destination" 

     change back to your app again...

third time，when you click "STEP1" again，it still show the Apple Maps but still have a old title "Satrt to Destination" and old "Route line"

if I use GoogleMap to instead，it have no this problem
Android and wp8 seem have no this problem too

**Comment by dpa99c** [comment]
I can replicate this issue and it does appear to be an issue with Apple Maps; if you press the "Clear" button in Apple Maps in between calls from the plugin, the issue doesn't occur. I'll investigate to see if there's a workaround.

**Comment by bau720123** [comment]
really thanks @dpa99c 
another question
is "Satrt to Destination" is a "Default" value in Apple Map when using latitude and longtitude

because of when in GoogleMap that it really show the real latitude and longtitude

**Comment by dpa99c** [comment]
"Start" and "Destination" are defaults being set by the plugin when using Apple Maps.

**Comment by bau720123** [comment]
thanks @dpa99c 
it really a good idea that you thought !!!
you are the best !!!

**Comment by dpa99c** [comment]
I tracked this issue (STEP 1, STEP2, STEP1) down to a race condition in the geocoding within the iOS native code of the plugin. It has been fixed in v2.9.4 of the plugin; this version also now displays the address for start/destination when they are given as longitude/latitude.

**Comment by bau720123** [comment]
problem solved in 2.9.5 version
thanks @dpa99c
